src/main/arm/createUiDefinition.json

```diff
@@ -237,6 +226,153 @@
                         }
                     }
                 ]
+            },
+            {
+                "name": "IHSConfig",
+                "label": "IBM HTTP Server Load Balancer",
+                "subLabel": {
+                    "preValidation": "Configure the IBM HTTP Server load balancer that routes client requests to IBM WebSphere cluster.",
+                    "postValidation": "Done"
+                },
+                "bladeTitle": "IHS configuration",
+                "elements": [
+                    {
+                        "name": "configIHS",
+                        "type": "Microsoft.Common.OptionsGroup",
+                        "label": "Configure an IBM HTTP Server?",
+                        "defaultValue": "Yes",
+                        "toolTip": "Select 'Yes' and provide required info to configure an IBM HTTP Server.",
```

Comment:
I think we should mention that if they select `No` they can not use this offer to add an HTTP server later. They will either need to set up a new cluster or add an HTTP Server manually.

Reply:
Good point. @git4rk Could you pls provide the improved tool tip message based on your thoughts above? Maybe you can also request your documentation team to review this message.

Reply:
@majguo Here is the suggested text (after reviewing from doc team):
Select 'Yes' and provide the required information to configure an IBM HTTP Server. If you select 'No', you will not be able to use this offer to add IBM HTTP Server later to an existing cluster. You either must set up a new cluster or add IBM HTTP Server manually.

Reply:
Thanks. Fixed in b6c4aa8.